src/orffinder/__main__.py

```diff
@@ -1,14 +1,16 @@
 from .orffinder import *
+import argparse
 
 if __name__ == "__main__":
-    if len(sys.argv) < 4:
-        exit("Usage: python3 orffinder.py reference_genomes.fasta query_sequences.fasta [best|all]")
-    genome, query = sys.argv[1], sys.argv[2]
-    runtypes = ["best", "all"]
-    runtype = sys.argv[3]
-    if runtype not in runtypes:
-        exit("Error: Unrecognised run type: " + runtype)
-    if runtype == "best":
-        OrfFinder(genome, query).bestOrf()
-    elif runtype == "all":
-        OrfFinder(genome, query).allGoodOrfs()
+    parser = argparse.ArgumentParser(description="Find open reading frames (ORFs) in the given sequence.")
+    parser.add_argument('reference_genome', type=str, help="""The path to a FASTA file containing the reference genome used to cross-reference found ORFs against using tBLASTn.""")
```

Comment:
"genome" should probably be "genome(s)"... though that might sound like you can provide multiple files.

Reply:
Yeah, that's the issue. We can easily have more than one reference genome here, and I guess this could work with the blast code, although it probably needs some changes.

Reply:
I'm going to leave it like this for now and will do a pull request that allows multiple fasta files. Should just simply merge them on read I guess.